Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1464,6 +1464,30 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
 
                 // Update the wheel's visibility
                 m_pVehicle->SetWheelVisibility((eWheelPosition)ucWheel, (ucStatus != DT_WHEEL_MISSING));
+
+                // restart wheels component visibility (fix #1556)
+                std::map<SString, SVehicleComponentData>::iterator iter = m_ComponentData.begin();
+
+                for (; iter != m_ComponentData.end(); iter++)
```

Comment:
Why not use a structured binding instead of iterators?

Reply:
I based it on a code snippet that does the same thing (fixes wheel visibility) but within the fixVehicle function. Wanted to maintain consistency throughout the code and the rest of the project.